<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Canvas</class>
 <widget class="QWidget" name="Canvas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
       <item>
        <widget class="QWidget" name="widget_tool" native="true">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,1">
             <property name="spacing">
              <number>10</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>20</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="minimumSize">
                    <size>
                     <width>75</width>
                     <height>25</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>25</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>功能节点&gt;&gt;</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>17</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <property name="spacing">
                    <number>30</number>
                   </property>
                   <item>
                    <widget class="QPushButton" name="ToolBtn_data">
                     <property name="minimumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="../../../icons.qrc">
                       <normaloff>:/pic/icons/func_数据输入icon.png</normaloff>:/pic/icons/func_数据输入icon.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="checkable">
                      <bool>false</bool>
                     </property>
                     <property name="checked">
                      <bool>false</bool>
                     </property>
                     <property name="autoRepeat">
                      <bool>false</bool>
                     </property>
                     <property name="autoExclusive">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="ToolBtn_Modeling">
                     <property name="minimumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="../../../icons.qrc">
                       <normaloff>:/pic/icons/func_代理模型icon.png</normaloff>:/pic/icons/func_代理模型icon.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="checkable">
                      <bool>false</bool>
                     </property>
                     <property name="autoRepeat">
                      <bool>false</bool>
                     </property>
                     <property name="autoExclusive">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="ToolBtn_Opt">
                     <property name="minimumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="../../../icons.qrc">
                       <normaloff>:/pic/icons/func_优化算法icon.png</normaloff>:/pic/icons/func_优化算法icon.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="ToolBtn_SA">
                     <property name="minimumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="../../../icons.qrc">
                       <normaloff>:/pic/icons/func_敏感性分析算法icon.png</normaloff>:/pic/icons/func_敏感性分析算法icon.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="ToolBtn_DOE">
                     <property name="minimumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="../../../icons.qrc">
                       <normaloff>:/pic/icons/func_.png</normaloff>:/pic/icons/func_.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>13</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>20</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="sizeConstraint">
                  <enum>QLayout::SetDefaultConstraint</enum>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="minimumSize">
                    <size>
                     <width>75</width>
                     <height>25</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>25</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>工具&gt;&gt;</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <spacer name="horizontalSpacer_7">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <property name="spacing">
                    <number>30</number>
                   </property>
                   <item>
                    <widget class="QPushButton" name="ToolBtn_Connect">
                     <property name="minimumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="../../../icons.qrc">
                       <normaloff>:/pic/icons/连接icon.png</normaloff>:/pic/icons/连接icon.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>35</width>
                       <height>35</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="ToolBtn_Delete">
                     <property name="minimumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="../../../icons.qrc">
                       <normaloff>:/pic/icons/删除icon.png</normaloff>:/pic/icons/删除icon.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>35</width>
                       <height>35</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="ToolBtn_Forbiden">
                     <property name="minimumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="../../../icons.qrc">
                       <normaloff>:/pic/icons/禁止icon.png</normaloff>:/pic/icons/禁止icon.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>35</width>
                       <height>35</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="ToolBtn_Activation">
                     <property name="minimumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>40</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="../../../icons.qrc">
                       <normaloff>:/pic/icons/解禁icon.png</normaloff>:/pic/icons/解禁icon.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>35</width>
                       <height>35</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_8">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="5,2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>300</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="minimumSize">
              <size>
               <width>2</width>
               <height>300</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>2</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="widget_log" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTextBrowser" name="textBrowser_log">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
